core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -49,6 +49,12 @@
               </div>
             </div>
           </div>
+          <div class="oppia-dashboard-topic-container" ng-if="$ctrl.showTopicSelector()">
+            <span class="oppia-dashboard-topic-container-label">Filter by Topics</span>
```

Comment:
I think the text should read "Filter by Topic" as we only allow one topic at a time. Do you mind also updating for the other topic selector?

Reply:
done